text/0000-bots-for-health.md

```diff
@@ -0,0 +1,93 @@
+*   Start date: 2020-11-04
+*   Scope: Collective
+*   RFC PR: <!-- leave this empty -->
+*   Implementation issue(s): <!-- leave this empty -->
+
+# Summary
+
+It’s becoming harder to handle the incoming stream of questions and issues, so I
+thought I’d open a discussion to talk about how to tackle that.
+
+Bots may help.
+
+This is a conversation starter: **I’m curious to hear what y’all think would
+(not) be useful**.
+
+## Motivation
+
+It’s becoming harder to handle the incoming stream of questions and issues, so I
+thought I’d open a discussion to talk about how to tackle that.
+
+I’m assuming this is coming from a couple of places:
+
+*   unified is becoming more popular
+*   MDX, recently react-markdown, and potentially more in the future, joined
+*   Discussions is an easy way for folks to ask questions
+
+Regardless of why, a different type of user is reaching unified (an exemplary
+question yesterday was: “why are there margins on my `<p>`”,
+[remarkjs/react-markdown#503](https://github.com/remarkjs/react-markdown/issues/503)).
+A lot of these questions are XY problems and result in maintainers acting like
+a rubber duck for three back-and-forths.
+
+I hope that automation can help deal with the flow of incoming questions so that
+maintainers can stay healthy.
+
+## Detailed design
+
+For a solution, here are a couple broad ideas:
+
+*   Improve docs on unifiedjs.com (important, but to be discussed somewhere
+    else)
+*   Improve contributing, support, and issue templates
+*   Shared saved responses (see
+    <https://github.com/electron/governance/tree/master/playbooks/responses>
+    and
+    <https://github.com/electron/electron/issues?q=is%3Aissue+is%3Aclosed>)
+*   Add automation/bots
+
+Specifically, for automation, I’m thinking:
+
+*   Enforce filled out issue/pr/discussion templates
+*   “Lint” issue checklists: if someone is on an old Node, respond and close
+    that those aren’t supported; or enforce that
```

Comment:
> if someone is on an old Node, respond and close
    that those aren’t supported;

Interesting :thinking: 
I like where this idea is going, what would be DX be like?
Would it be a checkbox? Would we try to parse the semver from the issue description?

Reply:
It’s interesting indeed, but now feels too structured for what issues are. Requiring folks to fill out Node/npm/etc is sometimes unrelated so I’d rather not list it all in the template.

Maybe a comment on the issue like this is more useful:

```markdown
Hi @mention, could you confirm that you are using the latest versions of
packages?
The latest version published of this project, `remark-rehype`, on npm is v1.2.3.

you can use `npm ls` in your terminal to find what packages and version you have
locally.

If you are instead on an older version, your issue may already be fixed.
...